### PR TITLE
🌐  Add French translation for `docs/fr/docs/deployment/index.md`

### PR DESCRIPTION
**Pull request by rjNemo**
🤗 

Here is the PR to translate the deployment intro.
See the french translation tracking issue [here](https://github.com/tiangolo/fastapi/issues/1972).

Thanks for the reviews

### CODE REVIEW

**Review by Smlep — APPROVED** [review]

**Review by arthurio — APPROVED** [review]
Looks good 👍🏻 🇫🇷 
However, I just checked https://github.com/tiangolo/fastapi/blob/master/docs/en/docs/deployment/index.md and looks like there is more to translate now?

**Review by arthurio — CHANGES_REQUESTED** [review]
I didn't see that it was out of sync with current english version, but the translation was great!

**Comment by rjNemo** [comment]
@arthurio I review the recent version as soon as possible.
Thanks for pointing it out

File: docs/fr/docs/deployment/index.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+Le déploiement d'une application **FastAPI** est relativement simple.
+
+## Que signifie le déploiement
+
+**Déployer** une application signifie effectuer les étapes nécessaires pour la rendre **disponible pour les
+utilisateurs**.
+
+Pour une **API Web**, cela implique normalement de la placer sur une **machine distante**, avec un **programme serveur**
+qui offre de bonnes performances, une bonne stabilité, _etc._, afin que vos **utilisateurs** puissent **accéder** à
```
Comment (arthurio) [inline docs/fr/docs/deployment/index.md:11]: Why `_ect._` instead of just `etc.`?

File: docs/fr/docs/deployment/index.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+Ceci contraste avec les étapes de **développement**, où vous êtes constamment en train de modifier le code, de le casser
+et de le réparer, d'arrêter et de redémarrer le serveur de développement, _etc._
+
+## Stratégies de déploiement
+
+Il existe plusieurs façons de procéder, en fonction de votre cas d'utilisation spécifique et des outils que vous
+utilisez.
+
+Vous pouvez **déployer un serveur** vous-même en utilisant une combinaison d'outils, vous pouvez utiliser un **service
+en nuage** qui fait une partie du travail pour vous, ou d'autres options possibles.
```
Comment (arthurio) [inline docs/fr/docs/deployment/index.md:23]: I think we should keep `"service cloud"`, WDYT?
Also, I would add `encore` at the end?

```suggestion
cloud** qui fait une partie du travail pour vous, ou encore d'autres options possibles.
```

**Review by arthurio — COMMENTED** [review]
2 questions, otherwise looks great 🤩

File: docs/fr/docs/deployment/index.md (hunk 1 above)
Comment (rjNemo) [reply docs/fr/docs/deployment/index.md:11]: I learned that Latin terms should be written in italics.

Comment (arthurio) [reply docs/fr/docs/deployment/index.md:11]: Thanks for the info 😃

**Review by arthurio — APPROVED** [review]
Great work! 💯

**Comment by tiangolo** [comment]
Awesome, thanks @rjNemo! :cake: 

And thanks for the reviews @arthurio and @Smlep! :coffee:

**Pull request merged**